src/pybind/mgr/dashboard/frontend/src/app/shared/components/submit-button/submit-button.component.ts

```diff
@@ -28,6 +28,7 @@ import * as _ from 'lodash';
 })
 export class SubmitButtonComponent implements OnInit {
   @Input() form: FormGroup;
+  @Input() formDir: NgForm;
```

Comment:
Is it possible to use ``form`` for Reactive forms AND Template driven forms? This would make it much easier for developers. Otherwise i would like to see some notes here to clarify the difference of the two attributes ``form`` and ``formDir``.